patches/minecraft/net/minecraft/entity/projectile/ProjectileHelper.java.patch

```diff
@@ -9,3 +9,19 @@
                    if (d0 == 0.0D) {
                       entity = entity1;
                       vector3d = vector3d1;
+@@ -123,10 +123,15 @@
+       }
+    }
+ 
++   @Deprecated
```

Comment:
You probably should add a comment so then this line becomes like:
```java
@Deprecated //Forge: Use the predicate version of getHandWidth instead
```

Reply:
Will do